repos/spack_repo/builtin/packages/py_torch/package.py

```diff
@@ -146,6 +146,7 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
     # Required dependencies
     depends_on("c", type="build")
     depends_on("cxx", type="build")
+    depends_on("binutils@2.36:", type="build")
```

Comment:
Still a bit concerned about this on macOS. I would be okay with this change if we add:
```suggestion
    depends_on("binutils@2.36:", when="platform=linux", type="build")
```